packages/cashier/build/constants.js

```diff
@@ -19,16 +19,14 @@ const IS_RELEASE = process.env.NODE_ENV === 'production' || process.env.NODE_ENV
 
 const ALIASES = {
     _common: path.resolve(__dirname, '../src/_common'),
-    Components: path.resolve(__dirname, '../src/Components'),
-    Config: path.resolve(__dirname, '../src/Config'),
-    Containers: path.resolve(__dirname, '../src/Containers'),
-    Constants: path.resolve(__dirname, '../src/Constants'),
+    Components: path.resolve(__dirname, '../src/components'),
+    Config: path.resolve(__dirname, '../src/config'),
+    Containers: path.resolve(__dirname, '../src/containers'),
+    Constants: path.resolve(__dirname, '../src/constants'),
     Images: path.resolve(__dirname, '../src/public/images'),
```

Comment:
can we add Pages: path.resolve(__dirname, '../src/pages') also as part of these changes ?